### PR TITLE
chapter6: create models, controller, view, validates, has_secure_pass…

### PR DESCRIPTION
**Pull request by giapbnv-0886**
…word with regular expression,...

### CODE REVIEW

**Comment by giapbnv-0886** [comment]
ready

**Comment by giapbnv-0886** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  before_save {self.email = email.downcase}
```
Comment (huuphuc93) [inline app/models/user.rb]:
```suggestion
  before_save {email.downcase!}
```

**Comment by giapbnv-0886** [comment]
ready

**Comment by huuphuc93** [comment]
merged

**Pull request merged**